### ISSUE TITLE
NcAppNavigationItem with actions doesn’t have actions selected in backwards tabbing

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud-libraries/nextcloud-vue#8143]
Steps to reproduce:
- Have an NcAppNavigationItem with actions defined in a list
- Backwards tab (shift + tab) through the list
- Actions are not focused

**Comment by ShGKme** [comment]
Confirmed
